requirements.txt

```diff
@@ -1,5 +1,5 @@
 pysnmp
 netaddr
-paramiko
+netmiko
 retrying
 six
```

Comment:
I think we can get rid of this dependency now. I don't remember calling it anywhere other than `CiscoSwitch`.